{"ray": {"sid": "b0afd46db2f245918ff59c3f68240f45", "uid": null, "qid": "09123f7e23b14e56b2ddc04ee2951622", "rid": null, "bars": {"default": {"percent": "100", "remaining": "0"}}, "messages": [{"type": "ERROR", "content": "process - failed executing: [09123f7e23b14e56b2ddc04ee2951622]\nTraceback (most recent call last):\n  File \"/home/youseef/anaconda3/envs/openfabric/lib/python3.8/site-packages/openfabric_pysdk/app/worker.py\", line 90, in process\n    output = execution_callback_function(data, ray, self.state)\n  File \"/home/youseef/openfabric/openfabric-ai-software-engineer/openfabric-ai-software-engineer/main.py\", line 70, in execute\n    context_model, qa_model = model_loading()\n  File \"/home/youseef/openfabric/openfabric-ai-software-engineer/openfabric-ai-software-engineer/main.py\", line 63, in model_loading\n    qa_model = QAModel()\n  File \"/home/youseef/openfabric/openfabric-ai-software-engineer/openfabric-ai-software-engineer/main.py\", line 42, in __init__\n    self.tokenizer = AutoTokenizer('google/flan-t5-base')\nTypeError: __init__() takes 1 positional argument but 2 were given\n", "created_at": "2023-12-14T01:12:23.140639"}], "status": "FAILED", "created_at": "2023-12-14T01:12:14.819371", "updated_at": "2023-12-14T01:12:23.156932", "finished": true}, "in": {"text": ["hello, openfabric"]}, "out": {}}